src/controllers/user.controller.ts

```diff
@@ -593,8 +593,10 @@ export const requestFriend = async (req: Request, res: Response) => {
   }
```

Comment:
We also need an endpoint to just remove the user from sentFriendRequests (if someone unrequests)